stories/components/panel-header.stories.tsx

```diff
@@ -39,29 +35,25 @@ export function WithModal() {
         <Button minimal icon="filter" />
         <Button minimal icon="plus" />
       </PanelHeader>
-      <Modal isOpen={isOpen} onRequestClose={close} width={400}>
-        <Modal.Header>Settings</Modal.Header>
-        <Modal.Body>
+      <Dialog
+        title="Settings"
+        icon="cog"
+        isOpen={isOpen}
+        onClose={close}
+        style={{ width: 400 }}
```

Comment:
What is the width when you don't specify it?

Reply:
it will take content width, so no need to add `style={{ width: 400 }}`